builtin/providers/docker/resource_docker_container_test.go

```diff
@@ -311,6 +327,12 @@ resource "docker_container" "foo" {
 	memory = 512
 	memory_swap = 2048
 	cpu_shares = 32
+
+	capabilities {
+		add= ["ALL"]
+		drop = ["SYS_ADMIN"]
+	}
+
```

Comment:
this is what i mean - I wasnt sure how to make a block that can not be repeated. alternatively I could change the resource so it is just two set of string lists. like
```
add-capabilities =  ["ALL"]
drop-capabilities = ["SYS_ADMIN"]
```
Instead of the above